### ISSUE TITLE
[DOC]: Version tag and Release notes

### ISSUE THREAD

**Issue by reefland** [Mintplex-Labs/anything-llm#1857]
### What would you like to see?

I know this project has been under rapid development.  However, it would be appreciated if a Semantic Versioning scheme could be put in place instead of just `latest` tag with Release Comments to help follow/learn/understand the hard work this community has contributed. 

Also consider publishing images to `ghcr.io` instead of just `docker.io`.

**Comment by timothycarambat** [comment]
We already publish to gchr!
https://github.com/Mintplex-Labs/anything-llm/pkgs/container/anything-llm
https://github.com/Mintplex-Labs/anything-llm/blob/36c5cd16c75fe9b6415c979d7e60e68083414b15/.github/workflows/build-and-push-image.yaml#L77

The version tag for sure would be nice to get to - we have yet to do this for the docker version 😓

**Comment by reefland** [comment]
Sorry, I forgot to include a reference to the `render` tag for ghcr.  I'm running under Kubernetes and the suggested image seems to use that

https://github.com/Mintplex-Labs/anything-llm/blob/36c5cd16c75fe9b6415c979d7e60e68083414b15/cloud-deployments/k8/manifest.yaml#L95

The `render` tag is not available on ghcr:
```
ERROR: ghcr.io/mintplex-labs/anything-llm:render: not found
```

**Comment by reefland** [comment]
Thank you for adding version notes, they look great! Very helpful.

Hopefully the semantic version tags are coming soon.

**Comment by timothycarambat** [comment]
Resolved by cc594d4e3e37a1b34716c9e8253e982e29964d7d
https://hub.docker.com/r/mintplexlabs/anythingllm/tags

tags labeled `1.1` and `1.1.1` for example - build on release of new versions on Github
